src/tracer.js

```diff
@@ -231,13 +231,23 @@ export default class Tracer {
             ctx.traceId = parent.traceId;
             ctx.spanId = Utils.getRandom64();
             ctx.parentId = parent.spanId;
-            ctx.flags = parent.flags;
-
             // reuse parent's baggage as we'll never change it
             ctx.baggage = parent.baggage;
+            ctx.flags = parent.flags;
 
-            parent.finalizeSampling();
-            ctx.finalizeSampling();
+            // Parent exists without a sampling decision
+            // An example of where this happens is in mobile clients where sampler parameters
+            // are not propagated
+            if (parent.flags & constants.DEFERRED_SAMPLING_MASK) {
+                ctx.flags ^=  constants.DEFERRED_SAMPLING_MASK;
+                if (this._sampler.isSampled(operationName, internalTags)) {
+                    ctx.flags |= constants.SAMPLED_MASK;
+                    internalTags[constants.DEFERRED_SAMPLING_TAG_KEY] = true;
+                }
+            } else {
+                parent.finalizeSampling();
+                ctx.finalizeSampling();
```

Comment:
I would like to consider whether "finalized" and "deferred" are actually the same thing.